integration/commands_administration_test.go

```diff
@@ -270,7 +269,7 @@ func TestListDatabases(t *testing.T) {
 			}
 			actual.TotalSize = 0
 
-			assert.EqualValues(t, tc.expected, actual)
+			assert.ElementsMatch(t, tc.expected.Databases, actual.Databases)
```

Comment:
I believe they should be sorted, just like in `listCollections`

Reply:
I saw CI sometimes failing, yep let's sort it as the order is not deterministic :see_no_evil:.

Reply:
I saw CI sometimes failing, yep let's sort it as the order is not deterministic :see_no_evil:.

Reply:
MongoDB's responses to `listDatabases` and `listCollections` are always sorted. We have a workaround for that DocumentDB problem in `listCollection` implementation. We should apply the same for `listDatabases`, not in test.

Reply:
* https://www.mongodb.com/docs/manual/reference/command/listDatabases/
* https://www.mongodb.com/docs/manual/reference/command/listCollections/

Hm, both are now explicitly documented as unsorted

Reply:
5d72de9d74a2e67070fc3fab996d6e34dff5a9fa

@noisersup Please check if we could simplify `listCollections` tests using `assert.ElementsMatch`, too

Reply:
I couldn't find any test that we could apply the same logic there.

Reply:
We use `AssertEqualDocumentsSlice` in this test, which ignores the order of fields of nested documents. The tests fail if `assert.ElementsMatch` is used, because of some fields being in different order.